components/VideoTable.vue

```diff
@@ -97,7 +160,15 @@ import _ from 'lodash'
             courseName,
           }
         })
-      }
+      },
+      itemsFilteredByTitleOrDescription() {
+        let isSearchTermInNameOrDescription = (item) => RegExp(this.search, 'i').test(item.name) || RegExp(this.search, 'i').test(item.description)
+        return this.search.length > 0 ? this.mungedVideos.filter(item => isSearchTermInNameOrDescription(item)) : this.mungedVideos;
```

Comment:
A possible refactor of this computed property:

```js
if(this.search.length == 0) { return this.mungedVideos }

let isSearchTermInNameOrDescription = (item) => RegExp(this.search, 'i').test(item.name) || RegExp(this.search, 'i').test(item.description)
return this.mungedVideos.filter(item => isSearchTermInNameOrDescription(item))
```

Reply:
Done